Updates to Thanos Migration guide (#931)
kubecost/docs: install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

## Migration process

All of these steps should be performed on Kubecost v1.108.1 on the Primary Kubecost cluster. Only at the end, will you upgrade all Kubecost clusters to Kubecost 2.0. The goal of this doc is to gradually migrate off Thanos (it is no longer supported in the Kubecost v2.0 Helm chart), towards Federated ETL, then finally Aggregator.

> Please upgrade to v1.108.1 on the Primary before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0

Reviewer: ```suggestion
Please upgrade to v1.108.1 on the Primary before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0.
```

## Migration process

All of these steps should be performed on Kubecost v1.108.1 on the primary Kubecost cluster. Only after completing the steps in this guide should you upgrade all Kubecost clusters to v2.0. The goal of this doc is to gradually migrate off Thanos, which is no longer supported in the Kubecost v2.0+ Helm chart, towards Federated ETL, then finally Aggregator.

Please upgrade your primary cluster to v1.108.1 before completing any of the following steps. v1.108.1 ships with various utilities necessary to complete the upgrade to v2.0. You can upgrade to this specific version using this following command:

```
helm upgrade --install kubecost \
  --repo https://kubecost.github.io/cost-analyzer/ cost-analyzer \

>   --namespace kubecost --create-namespace \

Reviewer: ```suggestion
  --namespace kubecost \
```

  --version 1.108.1

> ```

Reviewer 2: I've added this section showing how to upgrade your Kubecost to a specific version, can someone verify this is correct?

Reviewer 2: Not sure if they need to create the Kubecost namespace if it already exists

### Step 7: Upgrade your secondary clusters to build and push ETL data

> For this step, the secondary clusters DO NOT need to be on v2.0.  However, you must be running a version of Kubecost that supports Federated ETL.  >v1.99.0.

Reviewer: ```suggestion
For this step, the secondary clusters **DO NOT** need to be on v2.0. However, you must be running a version of Kubecost that supports Federated ETL (greater than v1.99.0).
```

Remove [Thanos configurations](https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/v1.108.1/cost-analyzer/values-thanos.yaml) from your primary cluster since it is no longer included in the Helm chart in 2.0. Your Kubecost installation is now configured to query the Aggregator for multi-cluster data, instead of Thanos.

### Step 9: Upgrade Primary Cluster to Kubecost 2.0

You can now upgrade the Primary Kubecost cluster to Kubecost 2.0 using your standard upgrade process. If upgrading via Helm, your upgrade command will look like:

```sh
helm upgrade kubecost cost-analyzer --repo https://kubecost.github.io/cost-analyzer/ \
  --namespace kubecost \
  -f values.yaml
```

### Step 10: Upgrade Secondary Clusters to Kubecost 2.0 (Optional)(Recommended)

> *While not absolutely necessary to upgrade secondary clusters to 2.0 immediately, we do recommend doing so as soon as possible

Reviewer: ```suggestion
{% hint style="info" %}
While not absolutely necessary to upgrade secondary clusters to 2.0 immediately, we recommend doing so as soon as possible.
{% endhint %}
```
